features/create_an_item.feature

```diff
@@ -8,5 +8,5 @@ Feature: Test post endpoint
     And I create a third Item
     When I request an index of Items
     Then I should see three items
-    And each item should have fields name, quality, sell_in
+    And each item should have fields name, quality, sell_in, description
     And the response should have a content type header type of `application/json`
```

Comment:
Can you add a new line at the end of the file? Might have been me, but that is what the little red minus thing is.

Reply:
done

Reply:
Definitely 👍 to this. In general, I would configure each editor you use for code to:
1. Trim whitespace on save, including the current line.
2. Add a trailing newline to the end of the file.